releases/release-5.2.0.md

```diff
@@ -0,0 +1,336 @@
+
+    + TiDB Lightning
+
+    - Support customized line terminators (except `\r` and `\n`) [#1297](https://github.com/pingcap/br/pull/1297)
+    - Support expression index and the index that depends on virtual generated columns [#1407](https://github.com/pingcap/br/pull/1407)
+
+    + Dumpling
+        - Support backing up MySQL compatible databases but does not support `START TRANSACTION ... WITH CONSISTENT SNAPSHOT` or`SHOW CREATE TABLE` [#311](https://github.com/pingcap/dumpling/pull/311)
+
+## Improvements
+
++ TiDB
+
+    - Support pushing down the built-in function `json_unquote()` to TiKV [#24415](https://github.com/pingcap/tidb/issues/24415)
+    - Support removing the `union` branch from the dual table [#25614](https://github.com/pingcap/tidb/pull/25614)
+    - Optimize the aggregate operator's cost factor [#25241](https://github.com/pingcap/tidb/pull/25241)
+    - Allow the MPP outer join to choose the build table based on the table row count [#25142](https://github.com/pingcap/tidb/pull/25142)
+    - Support balancing the MPP query workload among different TiFlash nodes based on Regions [#24724](https://github.com/pingcap/tidb/pull/24724)
+    - Support invalidating stale Regions in the cache after the MPP query is executed [#24432](https://github.com/pingcap/tidb/pull/24432)
+    - Improve the MySQL compatibility of the built-in function`str_to_date` for the format specifiers `%b/%M/%r/%T` [#25767](https://github.com/pingcap/tidb/pull/25767)
+    - Fix the issue that inconsistent binding caches might be created in multiple TiDB after recreating different bindings for the same query. [#26015](https://github.com/pingcap/tidb/pull/26015)
+    - Fix the issue that the existing bindings cannot be loaded into cache after upgrade [#23295](https://github.com/pingcap/tidb/pull/23295)
+    - Support ordering the result of `SHOW BINDINGS` by (`original_sql`, `update_time`) [#26139](https://github.com/pingcap/tidb/pull/26139)
+    - Improve the logic of query optimization when bindings exist, and reduce optimization times of a query [#26141](https://github.com/pingcap/tidb/pull/26141)
+    - Support completing the garbage collection automatically for the bindings in the "deleted" status [#26206](https://github.com/pingcap/tidb/pull/26206)
+    - Support showing whether a binding is used for query optimization in the result of `EXPLAIN VERBOSE` [#26930](https://github.com/pingcap/tidb/pull/26930)
+    - Add a new status variation `last_plan_binding_update_time` status variation to view the timestamp corresponding to the binding cache in the current TiDB instance. [#26340](https://github.com/pingcap/tidb/pull/26340)
+    - Support reporting an error when starting binding evolution or running `admin evolve bindings` to ban the baseline evolution (currently disabled in the on-premises TiDB version because it is an experimental feature) affecting other features. [#26333](https://github.com/pingcap/tidb/pull/26333)
```

Comment:
```suggestion
    - Support reporting an error when starting binding evolution or running `admin evolve bindings` to ban the baseline evolution (currently disabled in the on-premises TiDB version because it is an experimental feature) affecting other features [#26333](https://github.com/pingcap/tidb/pull/26333)
```